### PR TITLE
adding wrapper for validator

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Wrapper for validator

### CODE REVIEW

File: scripts/validator-wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+sed -i "1s/^/PROVIDER=wss:\/\/${domain}\n/" ./validator/.env
+controller_mnemonic=`cat generate-accounts/accounts/all/validator-${n}-controller | grep -Po '(?<="mnemonic":")(.*)(?=","p)'`
+stash_mnemonic=`cat generate-accounts/accounts/all/validator-${n}-stash | grep -Po '(?<="mnemonic":")(.*)(?=","p)'`
+echo $controller_mnemonic
+echo $stash_mnemonic
+sed -i '/STASH_ACCOUNT_MNEMONIC/d' ./validator/.env
+sed -i "2s|^|STASH_ACCOUNT_MNEMONIC=${stash_mnemonic}\n|" ./validator/.env
+sed -i '/CONTROLLER_ACCOUNT_MNEMONIC/d' ./validator/.env
+sed -i "3s|^|CONTROLLER_ACCOUNT_MNEMONIC=${controller_mnemonic}\n|" ./validator/.env
+docker-compose up add_validator
```
Comment (AndreiNavoichyk) [inline scripts/validator-wrapper.sh:49]: Have you tested its run from root folder?

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/validator-wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+fi
+
+# ============ Check arguments ============ 
+for arg in "$@"
+do
+  case $arg in
+    --domain=*)
+      export domain=`echo $arg | sed -e 's/^[^=]*=//g'`
+      ;;
+    --n*)
```
[the remark this answers is not in the document]
Comment (karthikeyan-cere) [reply scripts/validator-wrapper.sh:28]: renamed domain to id.

File: scripts/validator-wrapper.sh (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/validator-wrapper.sh:49]: No, Haven't tested it.

File: scripts/validator-wrapper.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndreiNavoichyk) [reply scripts/validator-wrapper.sh:28]: Not domain, `--n` to `--id` :-)

File: scripts/validator-wrapper.sh (hunk 1 above)
Comment (AndreiNavoichyk) [reply scripts/validator-wrapper.sh:49]: We can not merge anything without test. Please, do it.

File: scripts/validator-wrapper.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (karthikeyan-cere) [reply scripts/validator-wrapper.sh]: renamed the --n to --id

File: scripts/validator-wrapper.sh (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/validator-wrapper.sh:49]: Tested the scrip it is working from root folder

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**